server/channels/app/integration_action.go

```diff
@@ -460,6 +460,10 @@ func (a *App) OpenInteractiveDialog(request model.OpenDialogRequest) *model.AppE
 		return appErr
 	}
 
+	if dialogErr := request.IsValid(); dialogErr != nil {
+		return model.NewAppError("OpenInteractiveDialog", "app.open_interactive_dialog.invalid_request", nil, dialogErr.Error(), http.StatusBadRequest)
+	}
```

Comment:
Let's wrap the error
```suggestion
	if err := request.IsValid(); err != nil {
		return model.NewAppError("OpenInteractiveDialog", "app.open_interactive_dialog.invalid_request", nil, "", http.StatusBadRequest).Wrap(err)
	}
```

Reply:
Done. I wonder, if the wrapped information will be used, because for example in `server/channels/api4/integration_action_test.go` in the case `Should fail with same elements` it comes without wrapped error.